apps/weather_app/src/app/core/store/actions/city.actions.ts

```diff
@@ -0,0 +1,15 @@
+import { createAction, props } from '@ngrx/store';
+import { GeoCityData } from '../../models/cities.mode';
+
+export const saveEnteredCity = createAction(
+  '[Home Component] Save Current',
+  props<{ value: GeoCityData }>()
+);
+export const reset = createAction('[Home Component] Reset');
+
+export const loadInitialCity = createAction('[General] Load initial city');
+
+export const loadInitialCitySuccess = createAction(
+  '[Cities API] City Loaded Success',
+  props<{ value: GeoCityData }>()
+);
```

Comment:
seems that both actions are correspond to the same scope. Probably it is better to display it in the actions names